extensions/sar/README.md

```diff
@@ -30,12 +30,16 @@ To describe frame start and end times, use the [Date and Time Range fields](../.
 | sar:product_type            | string    | **REQUIRED.** The product type, for example `SSC`, `MGD`, or `SGC` |
 | sar:resolution_range        | number    | The range resolution, which is the maximum ability to distinguish two adjacent targets perpendicular to the flight path, in meters (m).  |
 | sar:resolution_azimuth      | number    | The azimuth resolution, which is the maximum ability to distinguish two adjacent targets parallel to the flight path, in meters (m).  |
-| sar:pixel_spacing_range     | number    | The range pixel spacing, which is the distance between adjacent pixels perpendicular to the flight path, in meters (m). Strongly RECOMMENDED to be specified for products of type `GRD`. |
-| sar:pixel_spacing_azimuth   | number    | The azimuth pixel spacing, which is the distance between adjacent pixels parallel to the flight path, in meters (m). Strongly RECOMMENDED to be specified for products of type `GRD`. |
+| sar:pixel_spacing_range     | number    | The range pixel spacing, which is the distance between adjacent pixels perpendicular to the flight path, in meters (m). Strongly RECOMMENDED to be specified for 'ground-range detected' products. |
+| sar:pixel_spacing_azimuth   | number    | The azimuth pixel spacing, which is the distance between adjacent pixels parallel to the flight path, in meters (m). Strongly RECOMMENDED to be specified for products of type 'ground-range detected' products. |
+| sar:product_pixel_spacing   | number    | The actual product pixel (column) spacing, in meters (m). While `sar:pixel_spacing_range` is given in in radar geometry, this property is in ground geometry. The two properties are usually mutually exlusive. |
+| sar:product_line_spacing    | number    | The actual product line (row) spacing, in meters (m). While `sar:pixel_spacing_azimuth` is given in in radar geometry, this property is in ground geometry.  The two properties are usually mutually exlusive. |
```

Comment:
i think just the sar:pixel_spacing_range and sar:pixel_spacing_azimuth is sufficient here, since they are mutually exclusive

Reply:
Maybe just the naming is off and we can make it more universal?

Reply:
`gsd`, that's what I was thinking. The pixel spacing for card4l product (which has to be in ground to be `analysis ready`) can be inferred from transform or explicitly defined in gsd. So they don't need to be defined elsewhere I think. pixel_spacing_range and pixel_spacing_azimuth just don't apply to these products.

Reply:
There might be special/rare case data are provided with different spacing in x and y. Not sure how STAC handles.

Reply:
It doesn't handle it at all. There's just one gsd for x and y so far. So do I understand correctly that we can remove sar:product_pixel_spacing / sar:product_line_spacing and just replace it with a single gsd entry? That would be nice, indeed.

Reply:
This is how we are defining GSD for the sensor.
https://www.ssec.wisc.edu/sose/tutor/ifov/define.html.

But, there are many definitions.

Reply:
resolution and sampling can be and often are for SAR different. My understanding of the face meaning of GSD is the same as product spacing used in card4l.

Reply:
So if I understood correctly, I'll remove the new product_*_spacing fields here and add gsd to the CARD4L SAR spec.

Reply:
@m-mohr missed your question above, but yes I think we gsd is sufficient for now.